program_repair.py

```diff
@@ -1,3 +1,39 @@
+def calc_index(input_string, line, column):
```

Comment:
It is still not 100% clear to me what this function does. Maybe you can put an example run in the comments? (an example of what goes in and out)

Reply:
I hope the latest commit makes it more clear.

The example is a `ParseException` at `location = e.line, e.column`:
https://github.com/Felienne/hedy/blob/3640a6fd4336b66d6fda893b3021b8705e82608f/hedy.py#L1742-L1749